plutus-script-utils/src/Plutus/Script/Utils/V2/Scripts.hs

```diff
@@ -0,0 +1,94 @@
+module Plutus.Script.Utils.V2.Scripts
+    ( -- * Script data hashes
+      PV2.Datum
+    , PV2.DatumHash
+    , PV2.Redeemer
+    , PV2.RedeemerHash
+    , P.datumHash
+    , P.redeemerHash
+    , P.dataHash
+    -- * Script hashes
+    , PV2.Validator
+    , PV2.ValidatorHash
+    , PV2.MintingPolicy
+    , PV2.MintingPolicyHash
+    , PV2.StakeValidator
+    , PV2.StakeValidatorHash
+    , validatorHash
+    , mintingPolicyHash
+    , stakeValidatorHash
+    , scriptHash
+    -- * Script utilities
+    , scriptCurrencySymbol
+    ) where
+
+import Cardano.Api qualified as Script
+import Cardano.Api.Shelley qualified as Script
+import Codec.Serialise (serialise)
+import Data.ByteString.Lazy qualified as BSL
+import Data.ByteString.Short qualified as SBS
+import Plutus.Script.Utils.V1.Scripts qualified as P
```

Comment:
Better to be consistent and `PV1`?

Sorry for being too pedantic.

Reply:
Not at all :) 

However, where does the `PV1` come from? `plutus-ledger-api` defines `V1`, not `PV1`.

Reply:
In other modules the imports are usually end with `qualified as PV1/PV2`.